client/src/state/DeckBuilderModule.ts

```diff
@@ -23,20 +24,27 @@ export const deckBuilderStore = vuexModule(rootStore, 'deckbuilder', {
     ) {
       state.decks = [];
       for (let initializer of init) {
-        const sideboard: CardColumn[] =
-            (<DraftCard[][]>Array(NUM_COLUMNS)).fill([]).map(() => []);
-        for (const card of initializer.pool) {
-          sideboard[Math.min(card.definition.cmc, sideboard.length - 1)]
-              .push(card);
+        const stored = localStorage.getItem(
+            `draft|${initializer.draftName}|${initializer.player.seatPosition}`);
```

Comment:
Suggest spinning the generation of the key off into a function, but you do you man